data/json/monsters/zed_acid.json

```
@@ -202,5 +202,164 @@
       "REVIVES",
       "PUSH_MON"
     ]
+  },
+  {
+    "id": "mon_zombie_acid_mortar",
+    "type": "MONSTER",
+    "name": { "str": "acid mortar zombie" },
+    "description": "This zombie's original face is gone, and in its place is a grotesque amalgamation of bio-mortar.  Bone and flesh extending from a swollen, cracked skull form a rounded muzzle, inside of which a constantly bubbling mass of acidic fluid boils.",
+    "copy-from": "mon_zombie_soldier",
+    "looks_like": "mon_zombie_soldier",
+    "diff": 20,
+    "harvest": "zombie_maybe_mil_bionics_acid",
```

Since these are now potential spitter upgrades and not military zeds
```suggestion
    "looks_like": "mon_zombie_corrosive",
    "diff": 20,
    "harvest": "zombie_acid",
```